src/style.css

```diff
@@ -1 +1,78 @@
-/* Styles go here */
+:root {
+  --blueHover: #00acdc;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.header__logo {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.nav__item {
+  margin-left: 20px;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__link {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+  position: relative;
+  display: flex;
+  align-items: center;
+  height: 60px;
+  color: #000;
+  text-decoration: none;
+  transition: color 0.3s;
+  cursor: pointer;
+}
+
+.nav__link:hover {
+  color: var(--blueHover);
+}
+
+.nav__link.is-active {
+  color: var(--blueHover);
+}
+
+.nav__link.is-active::after {
+  display: inline-block;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  content: "";
+  height: 4px;
+  width: 100%;
+  border-radius: 8px;
+  background-color: var(--blueHover);
+}
+
+@media screen and (min-width: 1200px) {
+  .header {
+    max-width: 1200px;
+    margin: auto;
+  }
```

Comment:
I would recommend do it for some new inner element, because now your header looks with some shades but it should not
<img width="365" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/6568745/9f2eceaf-db63-4dba-ae88-0d59822eacae">

Reply:
Done! I created inner div with class .header__container, and added media query for it!
![1](https://github.com/mate-academy/layout_moyo-header/assets/91736996/c9bdf5d3-79b7-480a-a03a-396b806a7176)
![2](https://github.com/mate-academy/layout_moyo-header/assets/91736996/406de16d-cf20-4ee6-bdca-c33fcdb4e8aa)